### ISSUE TITLE
Sandbox Folder

### ISSUE THREAD

**Issue by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#3080]
The game used to ship with a "Sandbox" folder. This folder was used to test mods and assets that weren't on the workshop yet and made it easier to manage mods without having them mixed in with the current bundles. Many modders have no idea this folder existed and don't ever think to create one and use it instead, so shipping it with the game would be very helpful.

**Comment by GazziFX** [comment]
Wow I used bundles folder of the map

**Comment by Jdance-Media** [comment]
> Wow I used bundles folder of the map

Yep, that's what I was doing, but it complicates things during projects. The Sandbox folder does me a big favor.
